### PR TITLE
added an api request to give us all the details we need in one

### PR DESCRIPTION
**Pull request by annieburns**
Hi Guys,

First, you will need to update your .env file.
Second, On the app.js file, I used this request which gives us all the info we need for a particular movie:

```
`const apiKey = process.env.API;
const movieId = 550;

/// ONCE WE ALREADY HAVE THE movieId (above)
// GET DETAILS BASED ON movieId
app.get('/', (req, res) => {
  request
    .get(`https://api.themoviedb.org/3/movie/${movieId}?api_key=${apiKey}&append_to_response=videos,credits`)
    .catch((err) => {
      res.send(err);
    })
    .then((movieDetails) => {
      res.send(movieDetails.text)
    });
}),`
```

**This is the response we get with the above, please note i hid some of the data which is expanded via the "+" :**
# **Questions are:  How should we clean this up?**

screen shot 2016-10-09 at 4 11 04 pm

### CODE REVIEW

File: app/app.js [hunk 1]
```diff
@@ -1,15 +1,48 @@
...
 
 app.use('/api/users', userRouter);
 
+////ANNIE'S API CALLS BEGIN HERE///////////////////////////////////////////
+const apiKey = process.env.API;
+const movieId = 550;
+
+/// ONCE WE ALREADY HAVE THE movieId (above)
+// GET DETAILS BASED ON movieId
+app.get('/', (req, res) => {
```
Comment (erikwithuhk) [inline app/app.js]: Can you rename this route to `'/api/trailers/:id` or something similar? That way, we can start building out the API.

File: app/app.js [hunk 2]
```diff
@@ -1,15 +1,48 @@
...
 const logger = require('morgan');
 const userRouter = require('./routes/userRouter.js');
+const request = require('superagent');
 
 const app = express();
 
+require('dotenv').config();
+process.env.ENV = process.env.ENV || 'dev';
+
+
```
Comment (erikwithuhk) [inline app/app.js]: Can you remove the extra line break here?

**Review by erikwithuhk — CHANGES_REQUESTED** [review]

File: app/app.js (hunk 2 above)
Comment (annieburns) [reply app/app.js]: yes!

File: app/app.js (hunk 1 above)
Comment (annieburns) [reply app/app.js]: done!

**Pull request closed without merging**